README.md

```diff
@@ -18,12 +18,12 @@ We will be building this app using Android Studio. This app will also consume th
  * Step 1 - Launch Android Studio -> New Project -> Create a new Android Project. Name your project AndroidPhotoApp.
 
 
-![minio_ANDROID2](https://github.com/minio/android-photo-app/blob/master/docs/screenshots/minio-ANDROID2.jpg?raw=true)
+![minio_ANDROID2](docs/screenshots/minio-ANDROID2.jpg?raw=true)
```

Comment:
I think we need full paths in these URLs. Because it will work in github but not when streamed from docs.minio.io.